server/strategy.go

```diff
@@ -79,12 +79,12 @@ func DecideOps(c *cke.Cluster, cs *cke.ClusterStatus, constraints *cke.Constrain
 	}
 
 	// 9. Uncordon nodes if nodes are cordoned by CKE.
-	if o := rebootUncordonOp(nf); o != nil {
+	if o := rebootUncordonOp(rqEntries, nf); o != nil {
 		return []cke.Operator{o}, cke.PhaseUncordonNodes
 	}
 
 	// 10. Reboot nodes if reboot request has been arrived to the reboot queue, and the number of unreachable nodes is less than a threshold.
-	if ops := rebootOps(c, reboot, nf); len(ops) > 0 {
+	if ops := rebootOps(c, rqEntries, newlyDrained, drainCompleted, drainTimedout, rebootDequeued, nf); len(ops) > 0 {
 		if len(nf.SSHNotConnectedNodes(nf.cluster.Nodes, true, true)) > constraints.RebootMaximumUnreachable {
```

Comment:
Is this condition correct?
Rebooting nodes cannot have ssh connections, so should they be excluded?

Reply:
Hmm... the behavior of the point seems to need more consideration.